packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts

```diff
@@ -46,6 +47,63 @@ const authProviderChoices = [
   },
 ];
 
+const conflictResolutionHanlderChoices = [
+  {
+    name: 'Auto Merge',
+    value: 'AUTOMERGE',
+  },
+  {
+    name: 'Optimistic Concurrency',
+    value: 'OPTIMISTIC_CONCURRENCY',
+  },
+  {
+    name: 'Custom Lambda',
+    value: 'LAMBDA',
+  },
+  {
+    name: 'Learn More',
+    value: 'Learn More',
```

Comment:
Should this have a key value that's all caps like the other options?

Reply:
This is an existing piece of code, we can clean it up with a different PR.